utils/aws_api.py

```diff
@@ -613,7 +613,12 @@ def get_cluster_vpc_id(self, account, route_tables=False):
         session = self.get_session(account['name'])
         sts = session.client('sts')
         role_arn = account['assume_role']
-        role_name = role_arn.split('/')[1]
+
+        try:
+            role_name = role_arn.split('/')[1]
```

Comment:
Now `role_name` won't be initialized and the code will break in the `sts.assume_role()` call below.